etl_manager/meta.py

```diff
@@ -192,24 +194,25 @@ def location(self, location):
             )
 
     @property
-    def personal_data(self):
-        return self._personal_data
-
-    @personal_data.setter
-    def personal_data(self, personal_data):
-        if personal_data is not None:
-            _validate_personal_data(personal_data)
-            self._personal_data = personal_data
-
-    @property
-    def special_category_data(self):
-        return self._special_category_data
-
-    @special_category_data.setter
-    def special_category_data(self, special_category_data):
-        if special_category_data is not None:
-            _validate_special_category_data(special_category_data)
-            self._special_category_data = special_category_data
+    def sensitivity(self):
+        return self._sensitivity
+
+    @sensitivity.setter
+    def sensitivity(self, sensitivity):
+        if not sensitivity:
+            column_sensitivities = {
+                column["sensitivity"]
+                for column in self.columns
+                if "sensitivity" in column
+            }
+            if column_sensitivities:
+                self._sensitivity = sorted(list(column_sensitivities))
+            else:
+                self._sensitivity = []
+        else:
+            _validate_table_sensitivity(sensitivity)
+            self._check_valid_table_sensitivity(sensitivity)
+            self._sensitivity = sorted(list(set(sensitivity)))
```

Comment:
I'm not sure if a user should ever be able to set this parameter? It is either derived from the column properties or if it is set it will throw an error if it doesn't align with column properties so it might be worth just having a function that recalculates it any time columns are changed?

Worth considering that there are occasions when you are setting the sensitivity (e.g. reading the a metadata json) but it might be worth considering if that should be treated as a "private" method rather than a "public" one.